starter-code/src/components/FilterableProductTable.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from "react";
+import "./FilterableProductTable.css";
+import SearchBar from "./SearchBar";
+import ProductTable from "./ProductTable";
+
+class FilterableProductTable extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchedProduct: "",
+      checkBox: false,
+      resultsProduct: []
+    };
+  }
+
+  updateSearch(event) {
+    const { value } = event.target;
+    this.setState({ searchedProduct: value });
+    console.log(this.state.searchedProduct);
+  }
+
+  searchProducts() {
+    const { searchedProduct, resultsProduct, checkBox } = this.state;
+    const { products } = this.props;
+
+    if (checkBox === true) {
+      var allResults = products.data.filter(oneProduct => {
+        return (
+          oneProduct.name.indexOf(searchedProduct) > -1 &&
+          oneProduct.stocked === false
```

Comment:
it should be `oneProduct.stocked === true` to display only the products available :)